releasetool/commands/start/ruby.py

```diff
@@ -193,6 +207,7 @@ def start() -> None:
     determine_package_name(ctx)
     determine_last_release(ctx)
     gather_changes(ctx)
+    edit_release_notes(ctx)
     update_changelog(ctx)
```

Comment:
Why is `update_changelog` called here, on line 211, and again on line 214?

Reply:
It was added by @busunkim96 in [](https://github.com/googleapis/releasetool/commit/28701b06377217bb9f424c2fa3f68b463b21224e#diff-d5854ea2c5b799bf7cd59ba2e1e7e415), at the same time that `releasetool.commands.common.edit_release_notes(ctx)` was removed.

Reply:
I'll remove it and update the PR, thanks @blowmage.